src/types/Firestore.ts

```diff
@@ -85,6 +85,7 @@ export interface FirestoreOrdered {
   statsOpenVenues?: Array<WithId<AnyVenue>>;
   subvenues?: WithId<AnyVenue>[];
   subvenueEvents?: WithId<VenueEvent>[];
+  marketingemails?: Array<string>;
```

Comment:
Not sure about the type of this entity, but I didn't dig into that.

#https://github.com/sparkletown/sparkle/pull/801/files#r555857316

Reply:
I wouldn't bother adding it back here personally. That will just confuse the fact that it is almost certainly unused.

```suggestion
```